UPGRADE-6.0.md

```diff
@@ -112,6 +112,8 @@ Messenger
  * Use of invalid options in Redis and AMQP connections now throws an error.
  * The signature of method `RetryStrategyInterface::isRetryable()` has been updated to `RetryStrategyInterface::isRetryable(Envelope $message, \Throwable $throwable = null)`.
  * The signature of method `RetryStrategyInterface::getWaitingTime()` has been updated to `RetryStrategyInterface::getWaitingTime(Envelope $message, \Throwable $throwable = null)`.
+ * Removed `prefetch_count` parameter in AMQP bridge.
+
```

Comment:
can be removed

Reply:
Hmm. I'm not sure to understand why this removal should not be documented here?

Reply:
Oh. the empty line!
Done